roles/openshift_rhv/tasks/build_vm_list.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Creates a dictionary for use with oVirt.vm-infra role
+# https://github.com/oVirt/ovirt-ansible-vm-infra
+- name: Create virtual machine list fact
```

Comment:
Why not add this support to `vm-infra` role?

Reply:
+1 That sounds like a good idea.

In addition, some of the DNS automation code would be helpful here.